style.css

```diff
@@ -0,0 +1,64 @@
+header {
+    text-align: center;
+}
+
+header > h1 {
+    width: 100vw;
+    text-align: center;
+}
+
+section {
+    width: 200px;
+    margin: 0 auto;
+    border: 1px solid #B9B7BD;
+    border-radius: 10px;
+    padding: 16px;
+}
+
+.screen{
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.screen > div {
+    background-color: #E8E8E8;
+    border: 1px solid #B9B7BD;
+    border-radius: 8px;
+    width: 85%;
+}
```

Comment:
suggestion: add `padding` in this `div` to avoid text content from being too close to the border